gbrl/common/compression.py

```
@@ -0,0 +1,612 @@
+    """
+    Tree compression for parametric actor (policy) networks.
+    
+    Compresses tree-based policy networks by optimizing action distribution accuracy.
+    Supports categorical, Gaussian, deterministic, and supervised learning modes.
+    
+    Attributes:
+        dist_type (str): Distribution type - 'categorical', 'gaussian', 'deterministic',
+            or 'supervised_learning'.
+    """
+    
+    def __init__(self, k: int, gradient_steps: int, dist_type: str, n_trees: int,
```

Ruff (0.14.14)

[warning] Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)